### PR TITLE
Fix `site/tree/parents` request for link dialogs

### LINKED ISSUE #6627: Link dialog: The request to `tree/parents` failed
Screenshot 2024-08-17 at 7 01 11 PM

The the sandbox…
1. Go to writer field page
2. Type in some text, use the link dialog to select the `Home` page, save
3. Reload
4. Select the linked word and open the link dialog
5. See the error message

### PR DESCRIPTION
**Pull request by afbora**
### Fixes
- Link dialog: The request to tree/parents failed #6627

### Breaking changes
None

## Ready?

- [x] In-code documentation (wherever needed)
- [ ] Unit tests for fixed bug/feature
- [x] Tests and CI checks all pass

### For review team

- [ ] Add lab and/or sandbox examples (wherever helpful)
- [ ] Add changes & docs to release notes draft in Notion

### CODE REVIEW

File: panel/src/components/Navigation/PageTree.vue [hunk 1]
```diff
@@ -77,11 +77,12 @@ export default {
 		},
 		async preselect(page) {
 			// get array of parent uuids/ids
-			const parents = await this.$panel.get("site/tree/parents", {
+			const response = await this.$panel.get("site/tree/parents", {
 				query: {
 					page
 				}
 			});
+			const parents = response.data;
```
Comment (afbora) [inline panel/src/components/Navigation/PageTree.vue:85]: Maybe more simple writing 🤔 

```suggestion
			const parents = await this.$panel.get("site/tree/parents", {
				query: {
					page
				}
			}).data;
```

Comment (distantnative) [reply panel/src/components/Navigation/PageTree.vue:85]: I think I like your other version better (the one you used in your commit). The `.data` at the end of the expression might be easier overlooked.

**Comment by distantnative** [comment]
@afbora this needs to point towards `release/4.4.0` as `develop-minor` already has 4.5 changes on it.

**Comment by afbora** [comment]
@distantnative Now, is it OK?

**Review by distantnative — APPROVED** [review]
Works now well for https://github.com/getkirby/kirby/issues/6627

**Pull request merged**